policy/compiler/compiler.go

```diff
@@ -145,19 +145,30 @@ func (ic *instanceCompiler) compile() (*model.Instance, *cel.Issues) {
 	if found {
 		ic.compileSelectors(selector.Ref, cinst)
 	}
-	rules, found := m.GetField("rules")
-	if found {
+	rules, rsfound := m.GetField("rules")
+	if rsfound {
 		ruleSet := ic.listValue(rules.Ref)
 		cinst.Rules = make([]model.Rule, len(ruleSet.Entries))
 		for i, rule := range ruleSet.Entries {
 			cinst.Rules[i] = ic.convertToRule(rule)
 		}
 	}
-	rule, found := m.GetField("rule")
-	if found {
+	rule, rfound := m.GetField("rule")
+	if rfound {
 		r := ic.convertToRule(rule.Ref)
 		cinst.Rules = []model.Rule{r}
 	}
+	if len(cinst.Rules) > ic.limits.RuleLimit {
```

Comment:
Either as part of this PR or a future on, please add an error report when both the `rule` and `rules` fields are specified.

Reply:
added.
was wondering if they were mutually exclusive.